test/utils/ast-utils.test.ts

```diff
@@ -35,10 +40,40 @@ describe('AST utils', () => {
       }
       `,
     ])(
-      "reutrns false for modules without a valid '@sentry/' import",
+      "returns false for modules without a valid '@sentry/' import",
       (code) => {
         expect(hasSentryContent(parseModule(code))).toBe(false);
       },
     );
   });
+
+  describe('hasSentryContentCjs', () => {
```

Comment:
l/m: Do you think it makes sense to think about the esm case? Maybe we can catch both by just looking for "@sentry/vite-plugin" in the file?

Reply:
Yeah, we can check for a (String)Literal starting with `@sentry/`. Tests seem to pass with this for esm and cjs. I'll change it.